packages/adapters-library/src/core/utils/unwrap.ts

```diff
@@ -47,8 +60,16 @@ export async function unwrap(
       blockNumber,
     )
 
-    token.tokens = unwrapExchangeRates?.tokens?.map(
-      (unwrappedTokenExchangeRate) => {
+    if (!unwrapExchangeRates?.tokens) {
+      return
+    }
+
+    if (!token.tokens) {
+      token.tokens = []
+    }
+
+    token.tokens.push(
```

Comment:
Needs to push instead of replace, as otherwise it removes the rewards which have already been added.